kartei.cls

```diff
@@ -327,21 +321,24 @@
   \fi%
   % Build Cover page before starting environment
   %
-\cohead{\color{CardHead}\theCardID}
-\lohead{\color{CardHead}\dasfach{#1}}
-\rohead{\color{CardHead}\derkommentar{#3}}
+\lohead{\theCardID} % VERTAUSCHT Bernd
+\cohead{\dasfach{#1}} % VERTAUSCHT Bernd
+%\cohead{\theCardID} % ORIGINAL Bernd
+%\lohead{\dasfach{#1}} % ORIGINAL Bernd
+\rohead{\derkommentar{#3}}
+\rofoot{Bernd S.E. (berndschelf@gmail.com)} %ERGÄNZT
+\lofoot{Analysis II V1.01} %ERGÄNZT
 \thispagestyle{scrheadings}
-~\vspace{2\baselineskip}\par{~\hfill \parbox[t]{.9\textwidth}{\centering \large #2}\hfill~}\vfill~
-\card@fronthook
+%~\vfill{~\hfill \parbox[t]{.9\textwidth}{#2}\hfill~}\vfill~
+{\parbox[t]{1\textwidth}{#2}}
+%{\parbox[t]{.9\textwidth}{#2}} ORIGINAL
 \newpage%
 }% Start os usual content block
-{
-\card@backhook
-% Build a warning if the page is too full (forcing a pagebreak)
+{% Build a warning if the page is too full (forcing a pagebreak)
  % assure the next card starts on an odd page by doublepagebreak
-\cohead{\color{CardHead}{\bfseries Achtung:} R\"uckseite von \theCardID\ ist zu voll.}%
```

Comment:
So bitte nicht, s.o., das bitte drinlassen, es gibt – meiner Meinung nach – keinen Grund, die rauszunehmen.

Reply:
Hmm, ich bin mir auch hier nicht sicher, warum sich da was geändert hat. Ich versuche gleich mal, ob sich sowas auch in GitHub ändern lässt.